Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,64 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "1.6.8"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "readline" => :build
+  depends_on "readline"
+
+  def install
+    args = []
+
+    args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
+    args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    args += std_cmake_args
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+    inreplace bin/"tarantoolctl", %r{\/usr\/local\/etc\/tarantool\/default\/tarantool}, "#{etc}/default/tarantool"
+
+    inreplace etc/"default/tarantool" do |s|
```

Comment:
You may need to do this `postinstall` because otherwise people who pour the bottles will have their `username` set as `brew` in this configuration file.